CHANGELOG.md

```diff
@@ -1,9 +1,234 @@
 # Changelog
 
+## v0.62.0
+
+This feature release of React Native includes many changes for the platform: it has been a lot of work, so please refer to the [blog post](ADD) for overlook of the main changes - like Flipper support and the new LogBox.
+
+As always, we’ve done our best to test this release at length, especially with this being such a large release. However, note that due to challenges caused by COVID-19 to people everywhere, we [the maintainers] may not have the bandwidth we’d like to have for quick follow-up on bugs and/or a patch release. When you run into an issue, please do try to provide a fix or a detailed ticket, but be prepared to revert to the version you were previously on to unblock yourself.
```

Comment:
@alloy just a super quick question, in this phrase 
> [the maintainers]

is this supposed to be a link to something or it was more like " we = the maintainers"?

Reply:
The latter 👍